src/elements/common/sub-header/Sort.js

```diff
@@ -6,12 +6,12 @@
 
 import React from 'react';
 import { FormattedMessage } from 'react-intl';
-import Button from 'box-react-ui/lib/components/button/Button';
-import DropdownMenu from 'box-react-ui/lib/components/dropdown-menu/DropdownMenu';
-import Menu from 'box-react-ui/lib/components/menu/Menu';
-import MenuItem from 'box-react-ui/lib/components/menu/MenuItem';
-import IconCheck from 'box-react-ui/lib/icons/general/IconCheck';
-import IconSort from 'box-react-ui/lib/icons/general/IconSort';
+import Button from 'components/button/Button';
```

Comment:
Can we also update these to import from index.js, such as `import Button from 'components/button';`? It looks like some files already do so.

Reply:
can do it in a followup commit, but we can for most iirc